src/styles/blocks/page.scss

```diff
@@ -0,0 +1,6 @@
+.page {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}

```

Comment:
add a wrapper to the calendar and move flex there

Reply:
wrapper is must to be inside the calendar?